contracts/test/accounts/Governor.sol

```diff
@@ -0,0 +1,148 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity 0.6.11;
+pragma experimental ABIEncoderV2;
+
+import { IERC20 } from "../../../modules/openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
+
+import { IMapleGlobals } from "../../interfaces/IMapleGlobals.sol"; 
+
+contract Governor {
+
+    /************************/
+    /*** Direct Functions ***/
+    /************************/
+    function mapleGlobals_setValidLoanFactory(address globals, address factory, bool valid)         external { IMapleGlobals(globals).setValidLoanFactory(factory, valid); }
+    function mapleGlobals_setValidPoolFactory(address globals, address factory, bool valid)         external { IMapleGlobals(globals).setValidPoolFactory(factory, valid); }
+    function mapleGlobals_setPriceOracle(address globals, address asset, address oracle)            external { IMapleGlobals(globals).setPriceOracle(asset, oracle); }
+    function mapleGlobals_setValidSubFactory(address globals, address fac, address sub, bool valid) external { IMapleGlobals(globals).setValidSubFactory(fac, sub, valid); }
+    function mapleGlobals_setCalc(address globals, address calc, bool valid)                        external { IMapleGlobals(globals).setCalc(calc, valid); }
```

Comment:
This style (or lack thereof) does not lend itself to easier readability, nor reviews.

![Screen Shot 2021-08-02 at 10 31 45 AM](https://user-images.githubusercontent.com/35537333/127878024-eeb69b84-dfb4-49d0-9333-2aa13d37657c.png)

```suggestion
    /************************/
    /*** Direct Functions ***/
    /************************/
    function mapleGlobals_setValidLoanFactory(address globals, address factory, bool valid) external {
        IMapleGlobals(globals).setValidLoanFactory(factory, valid);
    }

    function mapleGlobals_setValidPoolFactory(address globals, address factory, bool valid) external {
        IMapleGlobals(globals).setValidPoolFactory(factory, valid);
    }

    function mapleGlobals_setPriceOracle(address globals, address asset, address oracle) external {
        IMapleGlobals(globals).setPriceOracle(asset, oracle);
    }

    function mapleGlobals_setValidSubFactory(address globals, address factory, address subfactory, bool valid) external {
        IMapleGlobals(globals).setValidSubFactory(factory, subfactory, valid);
    }

    function mapleGlobals_setCalc(address globals, address calc, bool valid) external {
        IMapleGlobals(globals).setCalc(calc, valid);
    }

```